controllers/dogu_manager.go

```diff
@@ -32,18 +31,18 @@ var clientSetGetter = func(c *rest.Config) (kubernetes.Interface, error) {
 // The DoguManager creates, updates and deletes dogus
 type DoguManager struct {
 	scheme                    *runtime.Scheme
-	installManager            cloudogu.InstallManager
-	upgradeManager            cloudogu.UpgradeManager
-	deleteManager             cloudogu.DeleteManager
-	volumeManager             cloudogu.VolumeManager
-	ingressAnnotationsManager cloudogu.AdditionalIngressAnnotationsManager
-	supportManager            cloudogu.SupportManager
-	startStopManager          cloudogu.DoguStartStopManager
+	installManager            installManager
+	upgradeManager            upgradeManager
+	deleteManager             deleteManager
+	volumeManager             volumeManager
+	ingressAnnotationsManager additionalIngressAnnotationsManager
+	supportManager            supportManager
+	startStopManager          DoguStartStopManager
 	recorder                  record.EventRecorder
 }
 
 // NewDoguManager creates a new instance of DoguManager
-func NewDoguManager(client client.Client, ecosystemClient ecoSystem.EcoSystemV1Alpha1Interface, operatorConfig *config.OperatorConfig, eventRecorder record.EventRecorder) (*DoguManager, error) {
+func NewDoguManager(client client.Client, ecosystemClient ecoSystem.EcoSystemV2Interface, operatorConfig *config.OperatorConfig, eventRecorder record.EventRecorder) (*DoguManager, error) {
```

Comment:
🚫 **[checkstyle]** <funlen> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function 'NewDoguManager' is too long (74 > 60)
<!-- __reviewdog__:ChBhY2IxYzU2MzkyNjllZjJiEgpjaGVja3N0eWxl -->